### ISSUE TITLE
Zooming doesn't work when Karabiner is installed

### ISSUE THREAD

**Issue by 343max** [tekezo/Karabiner-Elements#421]
In System Preferences -> Accessibility -> Zoom is an option "Use scroll gesture with modifier to zoom". Zooming doesn't work when Karabiner is installed.

**Comment by jrolfs** [comment]
Just noticed this on macOS 10.12.1 (not beta)

**Comment by dusek** [comment]
Just adding that my modifier there is the default "^ Ctrl" (I am having the issue as well on macOS 10.12.1).

The only thing I mapped is CapsLock to Insert. Nothing with (from/to) Ctrl.

**Comment by skarekrow** [comment]
I'm also experiencing this. The exact setup @dusek is using except mine is Caps -> Backspace.

**Comment by seltzered** [comment]
Experiencing this too - here's what I experience (setup is swapping left ctrl & fn keys):

![lossyunt](https://cloud.githubusercontent.com/assets/297435/20243743/364faf70-a919-11e6-94b8-c8cd331699ed.gif)

**Comment by samyk** [comment]
Same here on 10.12.1.

**Comment by saagarjha** [comment]
Issue #388 already tracks this.

**Comment by tphares1** [comment]
This (and #388) are duplicates of #111.

**Comment by iiiGerardoiii** [comment]
Ctrl+Scroll doesn't work, but the Ctrl+Opt PIP zoom still works, you can use this as a temporary workaround.

Go to Accessibility > Zoom and select the Picture in Picture (PIP) zoom style, then click Options and check this checkbox:
![captura de pantalla 2016-11-15 01 42 04](https://cloud.githubusercontent.com/assets/6264194/20297199/dfdb2350-aad4-11e6-9752-8b3ef02e5ea0.png)

The zoom is not fullscreen, but it's something.

**Comment by 343max** [comment]
Thank you for fixing this!
